CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/objects/RawData.java

```diff
@@ -164,20 +164,19 @@ public boolean equals(Object obj) {
     @Override
     public String toString() {
         final StringBuilder repr = new StringBuilder();
-
-        if (StringUtils.isNotBlank(rawIdentifier)) {
-            repr.append(rawIdentifier);
-        }
+        repr.append(StringUtils.defaultString(rawIdentifier));
 
         if (StringUtils.isNotBlank(rawType)) {
-            repr.append("<").append(rawType).append(">");
+            repr.append("<");
+            repr.append(rawType);
+            repr.append(">");
```

Comment:
Unsure how I feel about this one. If you're wanting the appends on separate lines you could just move the `.append` to a new line but I don't think it needs to be called with `repr` for each append

Reply:
Reverting this, I find it easier to read but it could just be me

Reply:
you can still do each append on a different line. It just doesn't need `repr` called each time. i.e.
```
repr.append("<")
    .append(rawType)
    .append(">");
```